### PR TITLE
MBS-9085 enable configuration cache

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ USER_ID=$(shell id -u $(USER))
 docker=false
 infra?=
 ci?=false
-log_level?=--quiet
+log_level?=
```
Comment (dsvoronin) [inline Makefile:20]: To see that cache actually working; at least for a while

File: build-logic/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+org.gradle.unsafe.configuration-cache=true
+# TODO last two errors here: https://youtrack.jetbrains.com/issue/KT-43605
+#  should be ok to use it because properties are `java.rmi.server.hostname` and `kotlin.environment.keepalive`
+#  which should not change on local machine
+org.gradle.unsafe.configuration-cache-problems=fail
```
Comment (dsvoronin) [inline build-logic/gradle.properties:5]: should be `warn` here, but somehow it works for me;
Please try it, checking out the branch and running any task

File: docs/content/contributing/Troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`~/.gradle/gradle.properties`
+
+```properties
+org.gradle.unsafe.configuration-cache=false
+```
+
+**Why not opt-in?**
+
+Avito project has configuration cache disabled, just not all issues fixed for now.
```
Comment (eugene-krivobokov) [inline docs/content/contributing/Troubleshooting.md:16]:
```suggestion
Avito project has disabled configuration cache, just not all issues fixed for now.  
```
?

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/contributing/Troubleshooting.md (hunk 3 above)
Comment (dsvoronin) [reply docs/content/contributing/Troubleshooting.md:16]: i think both forms are pretty common
really not worth another CI runs)

Comment (eugene-krivobokov) [reply docs/content/contributing/Troubleshooting.md:16]: Hm, never seen it

**Pull request merged**